data/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+#
+# Do nothing if sync destination is not defined
+#
+if [ -z "$SYNC_DESTINATION" ]; then
+    exit 0
```

Comment:
It's the intended usage. I've mentioned in the README that it's a data image/container.